<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Play_Dialog</class>
 <widget class="QDialog" name="Play_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Play_Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pl_button">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_resume">
      <property name="text">
       <string>播放/恢复</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_pause">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_stop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="wd_menu" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pb_history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>历史记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_upload">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>上传视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_play">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_recorder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>录制视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>本地播放视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_download">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_comment">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>260</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>评论</string>
    </property>
   </widget>
   <widget class="QWidget" name="wd_content" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>721</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="play_lable">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_next">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>180</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>下一集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pre">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>220</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>上一集</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_time">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>400</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>03:00:00</string>
    </property>
   </widget>
   <widget class="QSlider" name="time_slider">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="cur_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_like">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>点赞</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_like">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_recommend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>猜猜你喜欢</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_play">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>播放量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_showplay">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>130</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cllike">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>300</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>取消点赞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_icon">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
